### PR TITLE
Added support for redirecting to the web

### PR DESCRIPTION
**Pull request by PieterKas**
Added option for redirecting to the web.

Notes: 

1. Re-using the PAR response - feels strange to have the "request_uri" parameter as the redirect URI - is that OK?
2. Added and example of redirecting to the web. Not entirely sure how control gets passed back to the "native" app/client or how much detail we should provide.

### CODE REVIEW

File: draft-parecki-oauth-first-party-apps.md [hunk 1]
```diff
@@ -288,6 +289,20 @@ For example,
       "authorization_code": "uY29tL2F1dGhlbnRpY"
     }
 
+### Redirect Response
+
+In the case where the authorization server wishes to interact with the
+user directly, it can return the redirect response. The authorization
+server may choose to interact directly with the user based on a risk
+assesment, the introduction of a new authentication method not supported
+in the applicatio, or to handle an exception flow like account recovery.
```
Comment (dteleguin) [inline draft-parecki-oauth-first-party-apps.md:298]: typo in "application"

File: draft-parecki-oauth-first-party-apps.md [hunk 2]
```diff
@@ -514,6 +529,17 @@ IANA has (TBD) registered the following values in the IANA "OAuth Authorization
...
+## Redirect to Authorization Server
+
+A user may be redirected to the Authorization Server to perfrom an account reset.
+
+* The Client collects username from the user.
+* The Client sends an Authorization Challenge Request ({{challenge-request}}) to the Authorization Challenge Endpoint ({{authorization-challenge-endpoint}}) including the username.
+* The Authorization Server verifies the username and determines that the account is locked and returns a Redirect message.
+* The Client parses the redirect message, opens a browser and redirects the user to the Authorization Server performing an OAuth 2.0 flow with PKCE.
+* The user resets their account by performing a multi-step authentication flow with the Authorization Server.
+* The Authorization Server issues an Authorizaton Code, which is exchanged for an access and refresh token bfore returning control to the Client.
```
Comment (dteleguin) [inline draft-parecki-oauth-first-party-apps.md:541]: typo in "before"

Comment (aaronpk) [inline draft-parecki-oauth-first-party-apps.md:541]:
```suggestion
* The Authorization Server issues an Authorizaton Code, which is exchanged for an access and refresh token before returning control to the Client.
```

File: draft-parecki-oauth-first-party-apps.md [hunk 3]
```diff
@@ -288,6 +289,20 @@ For example,
       "authorization_code": "uY29tL2F1dGhlbnRpY"
     }
 
+### Redirect Response
+
+In the case where the authorization server wishes to interact with the
+user directly, it can return the redirect response. The authorization
```
Comment (aaronpk) [inline draft-parecki-oauth-first-party-apps.md:295]:
```suggestion
user directly, it can return the `redirect` response. The authorization
```

**Pull request merged**